### DISCUSSION TITLE
Avoid to render arrow when property "START" or "END" doesn't exist.

### DISCUSSION THREAD

**Discussion by bismareeo** [Eliav2/react-xarrows#86] [Ideas]
![Screen Shot 2021-08-04 at 9 24 53 AM](https://user-images.githubusercontent.com/37048720/128188678-66b0fb7b-a640-4228-a1a1-6a9740d740da.png)
Good morning to everyone, Here we have an example of how the arrow is rendering between two boxes and it looks perfect, but what happened when one of the boxes is unmounted or deleted after rendering the arrow?
let me show you a screenshot: 
![Screen Shot 2021-08-04 at 9 28 41 AM](https://user-images.githubusercontent.com/37048720/128189282-73d3d299-744e-40ef-83df-c58d23f5e4b4.png)
Even if the box is unmounted or deleted, the arrow is still there, and what happens if you try to drag a little bit the box that was linked to the previous box. let me show you in the next screenshot:
![Screen Shot 2021-08-04 at 9 30 43 AM](https://user-images.githubusercontent.com/37048720/128189595-bf3e7a1c-3593-4422-a569-d6d9764f54ec.png).
I think now this is clear, I want to ask you. Is there a way to control to show or hide the arrow while a box is mounted or unmounted?. I am actually mounting and unmounting linked boxes and the arrows are becoming crazy. I am going to add a screenshot of how I am using the arrows v2.
![Screen Shot 2021-08-04 at 9 55 55 AM](https://user-images.githubusercontent.com/37048720/128193548-f702eb43-7109-4764-9d9b-1321b43b61ca.png)

**Comment by Eliav2** [comment]
OK, thank you @bismareeo! i Accept your suggestion. When a connected elem is not available the last remembered position would stay. 
Will add this into todos for next release.

**Reply by bismareeo** [reply]
Thank you @Eliav2, I am going to be waiting for the next release.

**Reply by bismareeo** [reply]
Dear @Eliav2 when are you planning to deploy the next release? any news?

**Reply by Eliav2** [reply]
I am on this lib in chunks, and for the next release I'm planning new modular architecture so extending this lib would be easier. so it will take some time.

**Reply by bismareeo** [reply]
Okay, @Eliav2 Thanks for letting me know that. I am going to be waiting for it. thanks in advance.
